packages/gatsby-plugin-fastify/src/utils/constants.ts

```diff
@@ -3,16 +3,31 @@ export const PATH_TO_FUNCTIONS = ".cache/functions";
 export const PATH_TO_PUBLIC = "public";
 export const PATH_TO_CACHE = ".cache";
 
+export const BUILD_HTML_STAGE = "build-html";
+export const BUILD_CSS_STAGE = "build-css";
+
 // Config Values
 
 export const CONFIG_FILE_NAME = "gatsby-plugin-fastify.json";
 export const CONFIG_FILE_PATH = PATH_TO_CACHE;
 
 // implementation values
 
-type Header = [string, string];
-export const IMMUTABLE_CACHING_HEADER: Header = [
-  `cache-control`,
-  `public, max-age=31536000, immutable`,
-];
-export const NEVER_CACHE_HEADER: Header = [`cache-control`, `public, max-age=0, must-revalidate`];
+export const IMMUTABLE_CACHING_HEADER = {
+  "cache-control": `public, max-age=31536000, immutable`,
+};
+export const NEVER_CACHE_HEADER = { "cache-control": `public, max-age=0, must-revalidate` };
```

Comment:
@moonmeister Did you mean sticking with objects as it is currently? I'm not sure which is better, I believe I stuck to the object because the existing code had it both ways and it was simpler to convert it all to objects than vice versa. I will re-visit as I refactor.

Reply:
I don't care which way it falls, let's just be consistent.